packages/fast-components-react-base/src/horizontal-overflow/horizontal-overflow.spec.tsx

```diff
@@ -257,18 +263,38 @@ describe("horizontal overflow", (): void => {
             ](0, 500, [])
         ).toBe(0);
     });
-    test("should have an `onLoad` method", () => {
+    test("should unmount appropriately by removing event listener", () => {
         const renderedWithImages: any = mount(
             <HorizontalOverflow managedClasses={managedClasses}>
                 {imageSet1}
             </HorizontalOverflow>
         );
 
+        renderedWithImages.unmount();
+        expect(renderedWithImages.removeEventListener).toBe(undefined);
```

Comment:
This does not do anything, I suggest you remove this test.

Reply:
resolved